docs/userguide/searchspace.md

```diff
@@ -1,46 +1,247 @@
+
+```python
+parameters = [
+    NumericalDiscreteParameter(name="p1", values=[0, 0.25, 0.5, 0.75, 1]),
+    NumericalDiscreteParameter(name="p2", values=[0, 0.25, 0.5, 0.75, 1]),
+]
+subspace = SubspaceDiscrete.from_simplex(max_sum=1.0, simplex_parameters=parameters)
+```
+
+~~~
+       p1    p2
+ 0   0.00  0.00
+ 1   0.00  0.25
+ 2   0.00  0.50
+ ..   ...   ...
+ 12  0.75  0.00
+ 13  0.75  0.25
+ 14  1.00  0.00
+ 
+ [15 rows x 2 columns]
+~~~
+
+### Representation of data within discrete subspaces
+
+Internally, discrete subspaces are represented by two dataframes, the *experimental* and the *computational* representation.
+
+The experimental representation (`exp_rep`) contains all parameters as they were provided upon the construction of the search space and viewed by the experimenter. The computational representation (`comp_rep`) contains a representation of parameters that is actually used for the internal calculation.
+
+In particular, the computational representation contains no more labels or constant columns. This happens e.g. for [`SubstanceParameter`](baybe.parameters.substance.SubstanceParameter) or [`CategoricalParameter`](baybe.parameters.categorical.CategoricalParameter).
+
+The following example demonstrates the difference:
+```python
+from baybe.parameters import NumericalDiscreteParameter, CategoricalParameter
+from baybe.searchspace import SearchSpace
+
+speed = CategoricalParameter("Speed", values=["slow", "normal", "fast"], encoding="INT")
+temperature = NumericalDiscreteParameter(name="Temperature", values=[90, 105])
+
+searchspace = SearchSpace.from_product(parameters=[speed, temperature])
+```
+
+~~~
+### Experimental representation ###
+     Speed  Temperature
+0    slow         90.0
+1    slow        105.0
+2  normal         90.0
+3  normal        105.0
+4    fast         90.0
+5    fast        105.0
+
+### Computational representation ###
+    Speed  Temperature
+0      0         90.0
+1      0        105.0
+2      1         90.0
+3      1        105.0
+4      2         90.0
+5      2        105.0
+~~~
+
+
+
+## `SubspaceContinuous`
+
+The `SubspaceContinuous` contains all the continuous parameters of a `SearchSpace`. There are different ways of constructing this subspace.
+
+### Using explicit bounds
+
+The [SubspaceContinuous.from_bounds](baybe.searchspace.continuous.SubspaceContinuous.from_bounds) method can be used to easily create a subspace representing a hyperrectangle.
+
+```python
+from baybe.parameters import NumericalContinuousParameter
+from baybe.searchspace import SubspaceContinuous
+
+parameters = [
+    NumericalContinuousParameter("param1", (0, 1)),
+    NumericalContinuousParameter("param2", (-1, 1)),
+]
+bounds = pd.DataFrame({p.name: p.bounds.to_tuple() for p in parameters})
+subspace = continuous = SubspaceContinuous.from_bounds(bounds)
+```
```

Comment:
it would be good to also show the resulting space (ie the pretty print version) here

Reply:
Added.